VS_Project/Pokerapp/Pokerapp/Players.cshtml

```diff
@@ -14,13 +14,30 @@
     string[] link = prelink.Split('/');
     int urlcode = Convert.ToInt32(link[4]);
 
-    //Return to login if players are lower than min_size
+    //Show message if players are lower than min_size to contact the admin
     int players = Convert.ToInt32(db.QueryValue("SELECT COUNT(Name) FROM Players WHERE Room_Code = @0", urlcode));
     int min_size = Convert.ToInt32(db.QueryValue("SELECT Tbl_Min_Size FROM Room WHERE Room_Code = @0", urlcode));
-    if (players < (min_size + 1))
+    bool cookies = false;
```

Comment:
cookies hoeven helemaal niet op de playerspagina? players kom je zonder cookies op

Reply:
Er staan cookies om te checken of het een admin is of niet. Als het de admin is zal de admin geredirect worden naar de AdminPage om spelers toe te voegen en als het niet een admin is komt er een melding in beeld.

Reply:
Waarom is dit? De admin moet ook naar players.cshtml kunnen gaan, net zoals spelers.

Reply:
sure